tools/encodings/GF 2016 Glyph Sets/glyphTypeSorting/EXPERT_composites_207.txt

```diff
@@ -0,0 +1,207 @@
+0x2153  ⅓ onethird
+0x2154  ⅔ twothirds
+0x215B  ⅛ oneeighth
+0x215C  ⅜ threeeighths
+0x215D  ⅝ fiveeighths
+0x215E  ⅞ seveneighths
+0x2191  ↑ upArrow
+0x2192  → rightArrow
+0x2193  ↓ downArrow
+          aacute.sc
```

Comment:
I don't think the smallcaps glyphs are composites; they need to be drawn (using 'powertools' like RMX, yes, but still, not composites... :)